### ISSUE TITLE
Admin Merchant Registration.

### ISSUE THREAD

**Issue by Mugerwa-Joseph** [mohamedsamara/mern-ecommerce#87]
I would suggest to give the admin the capacity to manually register merchants.

**Comment by mohamedsamara** [comment]
@Mugerwa-Joseph Definitely! I agree. It might not be on the top priority list but will need it at some point.

**Comment by kainatkiranrashid2** [comment]
Hello Mohamed Samara, First of thankyou for making such a wonderful project, I was looking into it, I downloaded the project and it is running perfectly, but I dont understand where to access admin panel, Pls respond

**Comment by Mugerwa-Joseph** [comment]
@kainatkiranrashid2 ,
Hope you managed to access the admin panel, if not, all you need is to create a user with the ROLE_ADMIN, and login with that user. The system will automatically redirect you to the admin panel.

Just in case you fail to create the user, you can modify the user straight from the database and change the role to ROLE_ADMIN.

Check the sample below:
![Admin](https://user-images.githubusercontent.com/19221105/168979423-6ad85f61-5ad8-4a34-bc6f-73b90e287e53.PNG)

**Comment by Mugerwa-Joseph** [comment]
Great.
Thanks @mohamedsamara
